src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -1338,6 +1338,48 @@ std::set<std::vector<element::Type>> jit_subtract_emitter::get_supported_precisi
     return {{element::f32, element::f32}, {element::i32, element::i32}};
 }
 
+/// EQUAL ///
+jit_equal_emitter::jit_equal_emitter(ov::intel_cpu::riscv64::jit_generator* host,
```

Comment:
Please move your implementations before `jit_exp_emitter` to align with alphabet order 😊